packages/form/src/component/DateTimePicker.stories.tsx

```diff
@@ -0,0 +1,141 @@
+import { action } from "@storybook/addon-actions";
+import { Story } from "@storybook/react";
+
+import { ISO8601DateAdapter } from "../form/adapter/date/ISO8601DateAdapter";
+import { ISODateTimeAdapter } from "../form/adapter/date/ISODateTimeAdapter";
+import { FormAdapter } from "../form/FormAdapter";
+import { toISO8601DateString } from "../util/date";
+import { DateTimePicker, DateTimePickerProps } from "./DateTimePicker";
+
+const Template: Story<DateTimePickerProps> = (args) => {
+  return <DateTimePicker {...args} />;
+};
+
+export const Standard = Template.bind({});
+Standard.args = {
+  onChange: (...args) => {
+    console.log("onChange", ...args);
+    action("onChange")(...args);
+  },
+};
+export const CustomFormat = Template.bind({});
+CustomFormat.args = { ...Standard.args, formatPattern: "dd.MM.yyyy, HH:mm:ss" };
+
+export const Prefilled = Template.bind({});
+Prefilled.args = { ...Standard.args, value: new Date() };
+
+export const MinDateToday = Template.bind({});
+MinDateToday.args = { ...Standard.args, minDate: new Date() };
+
+export const MaxDateToday = Template.bind({});
+MaxDateToday.args = { ...Standard.args, maxDate: new Date() };
+
+const ISO8601DateTemplate: Story<DateTimePickerProps> = (args) => {
+  return (
+    <FormAdapter date={ISO8601DateAdapter}>
```

Comment:
ISO8601 date format doesn't make any sense for date time picker (time is missing).
Can you remove all stories that are using the ISO8601DateTemplate template?